index.js

```diff
@@ -29,6 +29,7 @@ class RNAccountKit {
     readPhoneStateEnabled: true,
     receiveSMS: true,
     theme: {},
+    viewControllerMode: 'present' // for iOS only
```

Comment:
It would be nice to have this option included in the [Readme](https://github.com/underscopeio/react-native-facebook-account-kit/blob/master/README.MD) 

Also you can include a note maybe with the link of the issue it solves

Reply:
I updated README, could you please take a look?